R/''DeckGL.R

```diff
@@ -0,0 +1,19 @@
+# AUTO GENERATED FILE - DO NOT EDIT
+
+#' @export
+''DeckGL <- function(id=NULL, clickEvent=NULL, clickInfo=NULL, data=NULL, disableContext=NULL, dragEndEvent=NULL, dragEndInfo=NULL, dragStartEvent=NULL, dragStartInfo=NULL, enableEvents=NULL, hoverEvent=NULL, hoverInfo=NULL, mapboxKey=NULL, style=NULL, tooltip=NULL) {
```

Comment:
Hmm `''DeckGL` isn't going to work. Somehow that's coming from `--r-prefix ''` in the `build:py_and_r` package script. Either we should figure out how to fix this (get it back to just `deckGL` (somehow this PR is updating that as well?) and delete the files that reference `''DeckGL`) or, more likely, since we're not really supporting Dash for R at this point, just remove `--r-prefix ''` from that command and delete all the R artifacts (`R/`, `inst/deps/`, `man/`, `DESCRIPTION`, `NAMESPACE` entirely.

For that matter it'd be great to remove and `.gitignore` the Python build artifacts as well (all the contents of `dash_deck/` except for `__init__.py`, but that's not a big deal, we can leave that for now. We used to commit build artifacts so people could install directly from the repo, but in most projects we stopped doing that long ago.

Reply:
I deleted the R and man directories and reran `npm run build` I think the spurious `''` was as a result of @vogt31337 running build:py_and_r on windows. I ran the script from macos and it created the correct files.

I'd like to keep R support and built files in this PR as we're currently deploying our app by installing directly from the repo referencing this branch. I'll delete the built files in a new PR if you'd like

Reply:
Got it - that all makes sense. I know we have various issues with Windows, as none of our core devs use it so we rely on the community to help find and resolve bugs when running there (see eg https://github.com/plotly/dash/pull/2752)

Does that mean you're using this with Dash for R? That's great if so, just curious, we don't always know which of our packages people actually use!